upstairs/src/downstairs.rs

```diff
@@ -5837,6 +5859,7 @@ pub(crate) mod test {
     async fn reconcile_repair_workflow_1() {
         let mut ds = Downstairs::test_default();
         set_all_reconcile(&mut ds);
+        ds.reconcile_repair_needed = 2;
```

Comment:
Nit: move this to after the calls to `push_back` and make it
```rust
ds.reconcile_repair_needed = ds.reconcile_task_list.len()
```
to make the logic clear

Reply:
Made clear in 906d5b9